openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -182,7 +205,7 @@ def test_home_page_not_unified(self, user_type):
         """
         Verifies the course home tab when not unified.
         """
-        self.user = self.create_user_for_course(self.course, user_type)
+        self._create_test_user(user_type)
```

Comment:
This also got accidentally picked up in the rebase. The original is the correct logic.